Expensify/App
# Modify "Forgot" link to use ResendValidateCode

### Details
The "Forgot" link was using an API command that was sending out the wrong notification; let's change it to use the right one.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/160181

### Tests
Clicked "Forgot" link after entering an email in the sign in flow, verified that the right notification sent out and that I was able to set a new password with it.

### QA Steps
1. Create an account with an email address you have access to. I recommend using [EmailOnDeck](www.emailondeck.com)
2. Sign out and enter that email address.
3. Click "Forgot"
4. Check that email address for an email with the subject "Your magic sign in link for Expensify.cash"
5. Click on "here" in the email.
6. Set a new password.
7. Sign out and verify you can sign back in with this new password.

### Tested On


### Screenshots
Screenshots don't really show anything useful here
